drivers/sensor/vl53l0x/vl53l0x.c

```diff
@@ -42,16 +43,77 @@ LOG_MODULE_REGISTER(VL53L0X, CONFIG_SENSOR_LOG_LEVEL);
 
 struct vl53l0x_config {
 	struct i2c_dt_spec i2c;
+	struct gpio_dt_spec interrupt;
 	struct gpio_dt_spec xshut;
 };
 
 struct vl53l0x_data {
+	const struct device *dev;
 	bool started;
 	VL53L0X_Dev_t vl53l0x;
-	VL53L0X_RangingMeasurementData_t RangingMeasurementData;
+	VL53L0X_RangingMeasurementData_t measurement;
```

Comment:
This change doesn't look related to interrupt-gpios. Please extract to a separate commit.